source/_integrations/datadog.markdown

```diff
@@ -35,13 +35,7 @@ In the [Datadog Agent configuration](https://github.com/DataDog/datadog-agent/bl
 
 ## Configuration
 
-To use the `datadog` integration in your installation, add the following to your  {% term "`configuration.yaml`" %} file.
-{% include integrations/restart_ha_after_config_inclusion.md %}
-
-```yaml
-# Example configuration.yaml entry
-datadog:
-```
+{% include integrations/config_flow.md %}
 
 {% configuration %}
```

Comment:
for config flow, we use the [configuration_basic](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration_basic-block) tag. Please change to configuration_basic, endconfiguration_basic, and remove default, required and type entries. I you want to mention some of this information, It can be added to the description.